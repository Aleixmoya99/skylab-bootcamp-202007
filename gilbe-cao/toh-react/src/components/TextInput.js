import React from 'react';

function TextInput(props) {
	return (
		<label htmlFor={props.name}>
			name:
			<input
				name={props.name}
				type="text"
				placeholder={props.placeholder}
				value={props.value}
				onChange={props.onChange}
			/>
		</label>
	);
}

export default TextInput;
