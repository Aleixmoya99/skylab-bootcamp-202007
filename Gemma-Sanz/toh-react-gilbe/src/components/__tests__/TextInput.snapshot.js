import renderer from 'react-test-renderer';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import TextInput from '../TextInput';

describe('TextInput snapshot', () => {
    const textInputTree = renderer.create(
        <Router>
            <TextInput />
        </Router>
    );

    it('should match', () => {
        expect(textInputTree.toJSON()).toMatchSnapshot();
    });
});