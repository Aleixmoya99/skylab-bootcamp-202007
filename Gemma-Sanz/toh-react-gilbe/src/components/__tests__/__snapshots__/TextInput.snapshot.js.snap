// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput snapshot should match 1`] = `
<label>
  Name: 
  <input
    type="text"
  />
</label>
`;
